components/Form/index.js

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import { SubmitButton } from "../SubmitButton/index.js";
+
+const FormContainer = styled.form`
+  display: grid;
+  gap: 0.5rem;
+`;
+
+const Label = styled.label`
+  font-weight: bold;
+`;
+
+const Input = styled.input`
+  padding: 0.5rem;
+  font-size: inherit;
+  border: 3px solid black;
+  border-radius: 0.5rem;
+`;
+
+const Textarea = styled.textarea`
+  font-family: inherit;
+  border: 3px solid black;
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+`;
+
+const AddImageButton = styled.button`
+  background-color: black;
+  width: 10rem;
+  height: 5rem;
+  margin: auto;
+  margin-top: 5px;
+  padding: 0.8rem;
+  border-radius: 0.6rem;
+  color: white;
+  text-decoration: none;
+  font-weight: bold;
+  border: none;
+  font-size: inherit;
+`;
+
+
+
+export default function Form({ onSubmit, defaultData }) {
+    function handleSubmit(event) {
+        event.preventDefault();
+        const formData = new FormData(event.target);
+        const data = Object.fromEntries(formData);
+        onSubmit(data);
+}
+
+return (
+    <FormContainer onSubmit={handleSubmit}>
+
+      <AddImageButton type="submit">
+      Add Image
+      </AddImageButton>
```

Comment:
When you're not planning on making a button do something, best remove it. Put it back in when you're ready to fully implement it.

Reply:
My intention was to create a placeholder button as it was a task in the US1.